### DISCUSSION TITLE
Cannot do `gh pr view` with fine-grained access token

### DISCUSSION THREAD

**Discussion by flyx** [community/community#111324] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I have an access token that has read-write access to Pull Requests on my repository. I can use it to create a Pull Request via `gh pr create`.

However, When I try to view the Pull Request via `gh pr view`, I get

    GraphQL: Resource not accessible by personal access token (repository.pullRequests.nodes.0.projectCards.nodes)

I certainly shouldn't require *more* permissions to view a PR than to create it. I also don't understand what exactly this message tells me is missing – something related to Projects apparently, but I don't find any setting relating to Projects in fine-grained access tokens.

I tried settings some more permissions because that seems to have worked for other people with similar problems. However, I couldn't find a permission which, when enabled, allows me to do this.

I need a fine-grained access token because this is for a repository of an organization.

**Comment by vyas-n** [comment]
I ran into this issue running `gh pr view` using a fine-grained personal access token with the following repository permissions (no Org-level permissions):
```yaml
Read-only:
- Metadata
Read and Write:
- Actions
- Contents
- Pull requests
- Workflows
```

```shell
$ gh pr view
GraphQL: Resource not accessible by personal access token (repository.pullRequests.nodes.0.projectCards.nodes), Resource not accessible by personal access token (repository.pullRequests.nodes.0.statusCheckRollup.nodes.0.commit.statusCheckRollup)
```

**Comment by vyas-n** [comment]
This seems to be a related issue on the cli repo: https://github.com/cli/cli/issues/6274
